src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023-24 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0150" />
+    <field name="ListOrder"           mask="0x01" />
+    <field name="SelectWhileRunning"  mask="0x02" />
+  </bitmap>
+
+  <!-- Enum definitions for structures-->
```

Comment:
Please put the data types in the same order as the spec.  That makes reviewing _much_ easier.

Reply:
Done.